### DISCUSSION TITLE
How to trim image?

### DISCUSSION THREAD

**Discussion by DonHuskini** [cloudinary-community/next-cloudinary#192] [Q&A]
I have an image without background that has spaces around and I want to remove it. Cloudinary has a method `e_trim` which resolves it but I don't know how to use this method with this library. Can you point me how to do it?

I tried but doesn't work:
```
<CldImage width={150} height={150} src={src} alt={alt} effects={[{ trim: true }]} />
```

Ref: https://cloudinary.com/cookbook/trim_photo_background

**Comment by colbyfayock** [comment]
looks like this one isn't here yet - i'll add it! transferring this to an Issue

**Reply by colbyfayock** [reply]
now available via 4.9.0 https://github.com/colbyfayock/next-cloudinary/pull/195
